File: example/android/app/src/main/kotlin/com/flutter/twilio/voice_example/Application.java
package com.flutter.twilio.voice_example;

import com.google.firebase.FirebaseApp;

import io.flutter.app.FlutterApplication;
import io.flutter.plugin.common.PluginRegistry;

public class Application extends FlutterApplication implements PluginRegistry.PluginRegistrantCallback {
    @Override
    public void onCreate()
    {
        super.onCreate();
        FirebaseApp.initializeApp(this);
    }
    
    @Override
    public void registerWith(PluginRegistry registry) {
        AwesomePluginRegistrant.registerWith(registry);
        TwilioVoicePluginRegistrant.registerWith(registry);
    }
}

File: example/android/app/src/main/kotlin/app/krispcall/Application.kt
package app.krispcall

import android.content.Context
import com.google.firebase.FirebaseApp
import io.flutter.app.FlutterApplication
import io.flutter.plugin.common.PluginRegistry
import io.flutter.view.FlutterMain
import com.facebook.stetho.Stetho

class Application : FlutterApplication(), PluginRegistry.PluginRegistrantCallback {

    companion object {
        lateinit var context: Context
    }

    override fun onCreate() {
        super.onCreate()
        registerActivityLifecycleCallbacks(ActivityLifecycleListener)
        FirebaseApp.initializeApp(this)
        FlutterMain.startInitialization(this)
        context = applicationContext
        Stetho.initializeWithDefaults(this)

    }

    override fun registerWith(registry: PluginRegistry?) {
        TwilioVoicePluginRegistrant.registerWith(registry!!)
        SharedPreferencePluginRegistrant.registerWith(registry)
        InCallManagerPluginRegistrant.registerWith(registry)
        FlutterToastPluginRegistrant.registerWith(registry)
        AudioManagerPluginRegistrant.registerWith(registry)
        AwesomeNotificationPluginRegistrant.registerWith(registry)
    }
}
